web/src/user_group_edit.ts

```diff
@@ -4,7 +4,7 @@ import assert from "minimalistic-assert";
 import type * as tippy from "tippy.js";
 import * as z from "zod/mini";
 
-import render_confirm_delete_user from "../templates/confirm_dialog/confirm_delete_user.hbs";
+import render_confirm_delete_user from "../templates/confirm_dialog/confirm_deactivate_user_group.hbs";
```

Comment:
The name of what we call it when importing should always align with the filename. @shuklamaneesh23 please work on being disciplined about always self-reviewing your work before you open a PR -- just stepping back and reading this line, you should be able to notice this sort of thing (both Sahil's comment and mine) without a reviewer having to point it out to you.